pkg/common/constants.go

```diff
@@ -49,3 +49,8 @@ const (
 
 var SessionDataKey contextKey
 var SessionInternalKey contextKey
+
+//Regex
+const (
+	PROJECT_NAME_REGEX = `^[a-zA-Z][-a-zA-Z0-9]*[a-zA-Z0-9]$`
```

Comment:
This can be replaced with regex directly:
```go
var PrjNameRX = regexp.MustCompile(`^[a-zA-Z][-a-zA-Z0-9]*[a-zA-Z0-9]$`)
```
and used as:
```go
PrjNameRX.MatchString(project.Metadata.GetName())
```

No need of `MatchStringToRegx` func.

Reply:
Done @akshay196